server/@types/HdcStatus.ts

```diff
@@ -0,0 +1,17 @@
+class HdcStatus {
+  public constructor(
+    bookingId: string,
+    private readonly homeDetentionCurfewEndDate: string = null,
+    private readonly checksPassed: boolean = false,
```

Comment:
Dont think you need to specify the type as it is inferred by the default value

Reply:
No - the default value was added last minute during testing, so no it doesn't need the type. I will change

Reply:
Actually, it complains it could be type any without specifying it. Worth a check though.